app/values/tile_path.rb

```diff
@@ -48,13 +48,20 @@ def tileserver
     def valid?
       file_count = resource.decorate.try(:mosaic_file_count)
       return false unless file_count&.positive?
-      return false if resource.is_a?(ScannedMap) && resource.decorate.scanned_maps_count <= 1
       true
     end
 
     def mosaic?
-      return true unless resource.is_a?(RasterResource)
-      return true if resource.decorate.decorated_raster_resources.count.positive?
+      return true if resource.is_a?(RasterResource) && resource.decorate.raster_resources_count > 1
```

Comment:
I think we'd like to prefer `Wayfinder.for(resource).raster_resources_count` over `resource.decorate.raster_resources_count`